hack/local-setup.sh

```diff
@@ -154,6 +152,15 @@ deployKuadrant(){
 
   echo "Installing Kuadrant in ${clusterName}"
   ${KUSTOMIZE_BIN} build config/kuadrant | kubectl apply -f -
+
+  if [[ "${REDIS}" == true ]]; then
```

Comment:
What is this set to by default? I assume we really just want this all the time for a mctc setup no? Is there ever a case when we wouldn't want to have it setup with redis, at least for the time being until there is some othwer better option for multi cluster support? Having extra options will just make it more likely environments will drift and not be tested properly, unless it's really needed I'd just have it do one or the other (redis or no redis.)

Reply:
ok ill change to it being mandatory